src/index.html

```diff
@@ -126,9 +126,12 @@ <h3>imma lookin'</h3>
         </div>
         <button id="filterButton">apply filter</button>
       </section>
-      <section class="recipes-section recipes-section-js">
+      <section class="recipes-wrapper">
+        <p class="key hidden">Yellow Highlight: You have enough ingredients to cook this recipe!</p>
```

Comment:
I really like this idea. It immediately shows the user what recipes they can cook based on what's currently in their pantry.